src/webui/webapplication.cpp

```diff
@@ -334,7 +344,7 @@ void WebApplication::doProcessRequest()
         controller = session()->getAPIController(scope);
     if (!controller)
     {
-        if (scope == u"auth")
+        if (!isUsingApiKey && (scope == u"auth"))
```

Comment:
Shouldn't it return "Unauthorized" in case where inappropriate resource (e.g. "auth") is attempted to be accessed using API key?

Reply:
When auth is provided but lacks access to the resource, we'd want to use Forbidden.